store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreTest.java

```diff
@@ -754,5 +755,37 @@ public void shouldFederatedIfUserHasCorrectAuths() throws Exception {
         Assert.assertNull(x);
     }
 
+    @Test
+    public void shouldReturnSpecificGraphsFromCSVString() throws Exception {
```

Comment:
Please could you add some other combination of tests included ones that should error.

e.g
store.getGraphs(",mockGraphId2,mockGraphId4");
store.getGraphs("");
store.getGraphs("mockGraphId1");